pkg/processor/worker/worker.go

```diff
@@ -184,3 +185,9 @@ func (w *Worker) Subscribe(kind controlcommunication.ControlMessageKind, channel
 func (w *Worker) Unsubscribe(kind controlcommunication.ControlMessageKind, channel chan *controlcommunication.ControlMessage) error {
 	return w.runtime.GetControlMessageBroker().Unsubscribe(kind, channel)
 }
+
+func (w *Worker) WaitForStart() {
+}
+
+func (w *Worker) RunHandler() {
+}
```

Comment:
whats should be on them?

Reply:
@liranbg runHandler is something that is running on the background, currently we don't need it for worker specifically